src/experimentEnrollment.js

```diff
@@ -4,20 +4,21 @@ export default {
 
   enrollInExperiment(component, child, variationName) {
     if (variationName && child.props.name === variationName) {
-      component.exposedVariation = child;
+      component.selectedVariation = child;
     } else if (child.props.displayName === DEFAULT_EXPERIMENT_COMPONENT) {
       component.defaultComponent = child;
     }
     return component;
   },
-  
-  getExposedExperimentVariation(childrenComponents, variationName) {
+
+  getVariation(childrenComponents, variationName) {
+    // I don't understand what this line does.
     if (!childrenComponents.reduce) {
```

Comment:
We should use [`React.Children.count`](https://facebook.github.io/react/docs/top-level-api.html#react.children.count) for that check instead of checking for methods from `Array.prototype`.